src/main/remote-sync/helpers.ts

```diff
@@ -0,0 +1,86 @@
+import Store from 'electron-store';
+
+export const remoteSyncStore = new Store<{
+  lastFilesSyncAt?: string;
+  lastFoldersSyncAt?: string;
+}>();
+
+export const clearRemoteSyncStore = () => remoteSyncStore.clear();
+export function getLastFilesSyncAt(): Date | undefined {
+  const value = remoteSyncStore.get('lastFilesSyncAt');
+
+  if (!value) return undefined;
+
+  return new Date(value);
+}
+
+export function saveLastFilesSyncAt(date: Date, offsetMs: number): Date {
+  remoteSyncStore.set(
+    'lastFilesSyncAt',
+    new Date(date.getTime() - offsetMs).toISOString()
+  );
+  return date;
+}
+
+export function getLastFoldersSyncAt(): Date | undefined {
+  const value = remoteSyncStore.get('lastFoldersSyncAt');
+
+  if (!value) return undefined;
+
+  return new Date(value);
+}
+
+export function saveLastFoldersSyncAt(date: Date, offsetMs: number): Date {
+  remoteSyncStore.set(
+    'lastFoldersSyncAt',
+    new Date(date.getTime() - offsetMs).toISOString()
+  );
+  return date;
+}
+
+export type RemoteSyncedFile = {
+  id: number;
+  uuid?: string;
```

Comment:
This always exists

Reply:
uuid is not coming in the file response right now, can you add it?

Reply:
Done and deployed. Ref: https://github.com/internxt/drive-server-wip/commit/58250b1cedb87990e0d2e129ac57f8a9ffc9318a